ios/FluentUI/TextField/FluentTextField.swift

```diff
@@ -0,0 +1,313 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+import Combine
+
+@objc(MSFTextField)
+public final class FluentTextField: UIView, UITextFieldDelegate, TokenizedControlInternal {
+    public override func didMoveToWindow() {
+        super.didMoveToWindow()
+        tokenSet.update(fluentTheme)
+        updateTokenizedValues()
+    }
+
+    public typealias TokenSetKeyType = TextFieldTokenSet.Tokens
+    lazy public var tokenSet: TextFieldTokenSet = .init(state: { [weak self] in
+        return self?.state ?? .unfocused
+    })
+
+    /// UIImage used in the leading UIImageView. If this is nil, the leading UIImageView will be hidden.
+    @objc public var leadingImage: UIImage? {
+        didSet {
+            if let image = leadingImage {
+                leadingImageView.image = image
+                leadingImageContainerView.isHidden = false
+            } else {
+                leadingImageContainerView.isHidden = true
+            }
+        }
+    }
+
+    /// String used in the top label. If this is nil, the top label will be hidden.
+    @objc public var labelText: String? {
```

Comment:
is the labeltext coming from design? should it be more like title text?  labeltext and text seems too similar

Reply:
Would "topLabel" and "bottomLabel", or "label" and "input", or "title" and "input"?

FluentTextField.text seems a little odd. FluentTextField.title/input (or label/input) seems perhaps more clear, if that is the actual intention.

Reply:
The designs just say "label", but we can talk to them about changing it to title. Or we could change text to inputText.